src/ussd_app.js

```diff
@@ -230,16 +344,238 @@ go.app = function() {
                     " looking for?",
                 ].join("")),
                 choices: [
-                    new Choice("state_end", $("Nearest Clinic")),
-                    new Choice("state_end", $("MMC Clinic")),
-                    new Choice("state_end", $("HCT Clinic")),
+                    new Choice("nearest", $("Nearest Clinic")),
```

Comment:
@NicolaasDieBaas set both options to `true` in `make_clinic_search_params` utils